06-Plotting-Time-Series-Rasters-in-R.Rmd

```diff
@@ -85,49 +94,40 @@ This lesson is a part of a series of raster data in R lessons:
 * [Lesson 05 - Raster Time Series Data in R]({{ site.baseurl}}/R/Raster-Times-Series-Data-In-R/)
 * [Lesson 06 - Plot Raster Time Series Data in R Using RasterVis and LevelPlot]({{ site.baseurl}}/R/Plot-Raster-Times-Series-Data-In-R/)
 * [Lesson 07- Extract NDVI Summary Values from a Raster Time Series]({{ site.baseurl}}/R/Extract-NDVI-From-Rasters-In-R/)
-</div>
-
-```{r load-libraries }
-
-library(raster)
-library(rgdal)
-library(rasterVis)
-
-```
 
-#About the Time Series Data
-
-In this lesson, we are working with the same set of rasters taht we used in 
-[{{ site.baseurl}} /R/Raster-Times-Series-Data-In-R/ ](Lesson 05 - Time Series Rasters in R), 
-that were derived from the Landsat satellite - in `GeoTiff` format. Each
-raster covers the <a href="http://www.neoninc.org/science-design/field-sites/harvard-forest" target="_blank">NEON Harvard Forest field site</a>.
+###Sources of Additional Information
+* <a href="http://cran.r-project.org/web/packages/raster/raster.pdf" target="_blank"> Read more about the `raster` package in R.</a>
+* The `rasterVis` library can be used to create nicer plots of raster time
+series data! <a href="https://cran.r-project.org/web/packages/rasterVis/rasterVis.pdf"
+target="_blank">Learn more about the rasterVis package</a>.
 
+</div>
 
 ##Getting Started 
+In this lesson, we are working with the same set of rasters used in the
+[Raster Time Series Data in R ]({{ site.baseurl}} /R/Raster-Times-Series-Data-In-R/) 
+lesson. This data is derived from the Landsat satellite and stored in GeoTIFF
+format. Each raster covers the <a href="http://www.neoninc.org/science-design/field-sites/harvard-forest" target="_blank">NEON Harvard Forest field site</a>.  
+If you do not yet have this RasterStack, please create it now. 
 
-To begin, we will use the same raster stack that we used in the previous lesson.
+```{r load-libraries-data }
 
-
-```{r import-ndvi-rasters }
+library(raster)
+library(rgdal)
+library(rasterVis)
 
 # Create list of NDVI file paths
-NDVI_path <- "Landsat_NDVI/HARV/2011/ndvi"
+NDVI_path <- "Landsat_NDVI/HARV/2011/ndvi"  #assign path to object = cleaner code
 all_NDVI <- list.files(NDVI_path, full.names = TRUE, pattern = ".tif$")
 
```

Comment:
I can't seem to find this comment in the code but i'm fine with the "Cleaner code" notes but just want them above rather than next too when possible because the code chunks get wide.